whatis277/harvest
# harvest: Add dev overrides compose file

Adds a base compose file defining services shared by DEV and PROD

Adds an override compose file equivalent to the DEV environment

The PROD override compose file will be added in a follow up pull request

Testing instructions:
1. `dc up --build trellis bean pear`

1. Visit:

* bean: http://whatisbean.local
* pear: http://whatispear.local
* mailhog: http://localhost:8025

1. Ensure they all load fine
